### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dmitruz**
- [DEMO LINK](https://dmitruz.github.io/layout_calendar/)
- [TEST REPORT LINK](https://dmitruz.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ On hovering over a cell:
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://dmitruz.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://dmitruz.github.io/layout_calendar/report/html_report/)
```
Comment (loralevitska) [inline README.md:36]: Provide these links to PR description 
image

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: $square-size;
+    width: $square-size;
+    border: 1px solid black;
+    background-color: #eee;
```
Comment (loralevitska) [inline src/styles/blocks/calendar.scss]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffc0cb;
+      transform: translateY(-20px);
+    }
+  }
+}
```
Comment (loralevitska) [inline src/styles/blocks/calendar.scss]: Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) Fix all cases

**Review by loralevitska — CHANGES_REQUESTED** [review]
- implement these 2 points 
image

**Comment by dmitruz** [comment]
add chages and updated Demolink

File: src/styles/utils/_functions.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+@function get-margin-size($number-of-day) {
+  @return (($square-size * $number-of-day) + ($gap-size * $number-of-day));
+}
```
Comment (loralevitska) [inline src/styles/utils/_functions.scss:3]: Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**